packages/react-native/Libraries/AppDelegate/RCTAppDelegate.h

```diff
@@ -54,61 +56,13 @@ NS_ASSUME_NONNULL_BEGIN
  *   - (id<RCTTurboModule>)getModuleInstanceFromClass:(Class)moduleClass
  */
-@interface RCTAppDelegate : UIResponder <UIApplicationDelegate, UISceneDelegate, RCTBridgeDelegate>
+@interface RCTAppDelegate : UIResponder <UIApplicationDelegate, UISceneDelegate, RCTRootViewFactoryDelegate>
 
 /// The window object, used to render the UViewControllers
-@property (nonatomic, strong, nonnull) UIWindow *window;
-@property (nonatomic, strong, nullable) RCTBridge *bridge;
-@property (nonatomic, strong, nullable) NSString *moduleName;
-@property (nonatomic, strong, nullable) NSDictionary *initialProps;
-
-/**
- * It creates a `RCTBridge` using a delegate and some launch options.
- * By default, it is invoked passing `self` as a delegate.
- * You can override this function to customize the logic that creates the RCTBridge
- *
- * @parameter: delegate - an object that implements the `RCTBridgeDelegate` protocol.
- * @parameter: launchOptions - a dictionary with a set of options.
- *
- * @returns: a newly created instance of RCTBridge.
- */
-- (RCTBridge *)createBridgeWithDelegate:(id<RCTBridgeDelegate>)delegate launchOptions:(NSDictionary *)launchOptions;
-
-/**
- * It creates a `UIView` starting from a bridge, a module name and a set of initial properties.
- * By default, it is invoked using the bridge created by `createBridgeWithDelegate:launchOptions` and
- * the name in the `self.moduleName` variable.
- * You can override this function to customize the logic that creates the Root View.
- *
- * @parameter: bridge - an instance of the `RCTBridge` object.
- * @parameter: moduleName - the name of the app, used by Metro to resolve the module.
- * @parameter: initProps - a set of initial properties.
- *
- * @returns: a UIView properly configured with a bridge for React Native.
- */
-- (UIView *)createRootViewWithBridge:(RCTBridge *)bridge
-                          moduleName:(NSString *)moduleName
-                           initProps:(NSDictionary *)initProps;
-/**
- * This method can be used to customize the rootView that is passed to React Native.
- * A typical example is to override this method in the AppDelegate to change the background color.
- * To achieve this, add in your `AppDelegate.mm`:
- * ```
- * - (void)customizeRootView:(RCTRootView *)rootView
- * {
- *   rootView.backgroundColor = [UIColor colorWithDynamicProvider:^UIColor *(UITraitCollection *traitCollection) {
- *     if ([traitCollection userInterfaceStyle] == UIUserInterfaceStyleDark) {
- *       return [UIColor blackColor];
- *     } else {
- *       return [UIColor whiteColor];
- *     }
- *   }];
- * }
- * ```
- *
- * @parameter: rootView - The root view to customize.
- */
-- (void)customizeRootView:(RCTRootView *)rootView;
+@property (nonatomic, strong) UIWindow *window;
+@property (nonatomic, strong) NSString *moduleName;
+@property (nonatomic, strong) NSDictionary *initialProps;
+@property (nonatomic, strong) RCTRootViewFactory* rootViewFactory;
```

Comment:
why we lost the nullability annotation here? 🤔

Reply:
Fixed